validator/client/grpc-api/grpc_validator_client.go

```diff
@@ -146,3 +146,8 @@ func NewGrpcValidatorClient(cc grpc.ClientConnInterface) iface.ValidatorClient {
 func (c *grpcValidatorClient) StartEventStream(context.Context) error {
 	return nil
 }
+
+// EventStreamIsRunning returns a dummy value for gRPC
+func (c *grpcValidatorClient) EventStreamIsRunning() bool {
+	return true
```

Comment:
we should have a log at least for the dummy value.. same with the above

Reply:
I think it would actually be better to panic because these functions should never be called